spec/models/suspended_email_spec.rb

```diff
@@ -23,7 +23,7 @@
       it 'returns nil' do
         email = 'not_blocked@example.com'
 
-        expect(SuspendedEmail.find_with_email(email)).to be_nil
+        expect(SuspendedEmail.find_with_email(email)&.email_address).to be_nil
```

Comment:
I think this whole thing is just nil
```suggestion
        expect(SuspendedEmail.find_with_email(email)).to be_nil
```